data_prep/nvidia.py

```diff
@@ -70,16 +70,14 @@ def __getitem__(self, idx):
             'timestamp': np.array(frame["index"]).astype('datetime64[ns]').astype(np.float64),
         }
 
-        target_values = frame["steering_angle"]
+        target_values = np.array(frame["steering_angle"], dtype=np.float32)
```

Comment:
I asked a question regarding the change
https://discord.com/channels/1227217445517922406/1227217449838182435/1248257517880999967